lia-core/src/main/java/com/lithium/community/android/auth/LiLoginActivity.java

```diff
@@ -117,6 +117,7 @@ public void onCreate(Bundle savedInstanceBundle) {
                 webSettings.setJavaScriptEnabled(true);
                 webSettings.setCacheMode(WebSettings.LOAD_NO_CACHE);
                 webSettings.setAppCacheEnabled(false);
+                webSettings.setUserAgentString("Lithium-User:".concat(System.getProperty("http.agent")));
```

Comment:
use a string constant?

Reply:
It is not used anywhere else in this exact form, there is one other place where user agent is set explicitly, but that just does a  'System.getProperty("http.agent")' which is tied to some LSI logic, Can't change that.

Reply:
Just don't like hard-coded strings in code.. its nit-picky.. can ignore